modules/indexing/src/test/java/org/apache/ignite/internal/processors/cache/ttl/CacheTtlAbstractSelfTest.java

```diff
@@ -144,7 +144,7 @@ public void testDefaultTimeToLiveLoadCache() throws Exception {
 
         checkSizeBeforeLive(SIZE);
 
-        Thread.sleep(DEFAULT_TIME_TO_LIVE + 500);
+        Thread.sleep(DEFAULT_TIME_TO_LIVE + 1000);
```

Comment:
How it's related to patch?
Can we introduce some static final field instead of hardcoded value?

Reply:
These tests are flacky by their nature.
`checkSizeBeforeLive` become slightly faster (~20 ms because of removed check) and tests become more flacky.
So, added the cheapes fix (+500ms).